handlers_test.go

```
@@ -647,49 +648,99 @@ func TestIssueEdgeCases(t *testing.T) {
+		},
 	}
 
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			// Create a test model
-			m := model{
-				appModel: appModel{},
-			}
-			field := tt.initialValue
+	updated, _ := handleBackspace(m)
+	got := updated.(model).appModel.searchQuery
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)